### PR TITLE
NEZ Sensor EH to HH change + Datalogger bank Swap + New loc code to 12

### PR DESCRIPTION
**Pull request by salichon**

### CODE REVIEW

**Review by mnaguit — CHANGES_REQUESTED** [review]
Thanks @salichon . Here, a few suggested edits:
- streams at LC 12 (True for axial flag in `install/streams.csv`)
- is there a need to update the orientation of the sensor at LC 12? (in `install/sensors.csv`)

File: install/streams.csv [hunk 1]
```diff
@@ -1417,8 +1417,9 @@ NEZ,10,,,100,true,false,false,2010-06-07T23:30:01Z,2014-03-20T00:00:00Z
 NEZ,10,E,H,100,true,false,false,2010-02-21T00:00:02Z,2010-06-02T05:59:59Z
 NEZ,10,E,H,100,false,false,false,2010-06-02T06:00:00Z,2010-06-07T23:30:00Z
 NEZ,10,E,H,100,false,false,false,2014-03-20T00:01:00Z,2014-05-05T00:00:00Z
-NEZ,10,E,H,100,false,false,false,2020-01-31T05:44:00Z,9999-01-01T00:00:00Z
+NEZ,10,E,H,100,false,false,false,2020-01-31T05:44:00Z,2023-02-28T22:51:00Z
 NEZ,11,E,H,100,true,false,false,2010-06-07T23:35:00Z,2020-01-30T05:08:00Z
+NEZ,12,H,H,100,false,false,false,2023-02-28T22:51:00Z,9999-01-01T00:00:00Z
```
Comment (CBurton90) [inline install/streams.csv:1422]: `NEZ,12,H,H,100,true,false,false,2023-02-28T22:51:00Z,9999-01-01T00:00:00Z`

**Review by CBurton90 — CHANGES_REQUESTED** [review]

**Comment by salichon** [comment]
thanks @CBurton90 @mnaguit 
corrected PR 
The aZimuth of NEZ 12 will come later -we can track this in the ticket  when necessary  - iis it ok ?

**Review by mnaguit — APPROVED** [review]
All good Sir! Thanks! 👍

**Review by CBurton90 — APPROVED** [review]

**Comment by salichon** [comment]
Will  uadte azimuth when known :)

**Pull request merged**